ipi/engine/forces.py

```diff
@@ -301,6 +307,8 @@ def __init__(
               will be weighted by this factor.
            name: The name of the forcefield.
            mts_weights: Weight of forcefield at each mts level.
+           force_extras: A list of properties that should be treated as physical quantities,
+              converted to numpy arrays and treated with ring polymer contraction
```

Comment:
Here a description how to treat it with RP-contract would be good. I can amend this comment in a moment.